packages/components/src/app.tsx

```diff
@@ -81,17 +26,7 @@ export default function LiteflowNFTApp({
 
   return (
     <ErrorBoundary>
-      <ChakraProvider theme={theme}>
-        <ApolloProvider client={client}>
-          <Web3ReactProvider getLibrary={web3Provider}>
-            <CookiesProvider cookies={ssr ? new Cookies(cookies) : undefined}>
-              <LiteflowProvider endpoint={endpointUri}>
-                <Session {...sessionProps}>{children}</Session>
-              </LiteflowProvider>
-            </CookiesProvider>
-          </Web3ReactProvider>
-        </ApolloProvider>
```

Comment:
All these providers have been moved to the application. This way the application has control on the version or even the provider for the app. Also the application now controls the authentication thus requires the cookies and the web3 provider.

With this change, more logic is in the app (but also more control) and the component is only responsible for the theming (which may endup moving to the app too)